client/packages/system/src/Encounter/DetailView/SidePanel.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { FC, useState } from 'react';
+import {
+  DetailPanelSection,
+  Grid,
+  PanelLabel,
+  PanelField,
+  PanelRow,
+  DetailPanelPortal,
+  useTranslation,
+  useFormatDateTime,
+  TextArea,
+  Link,
+  RouteBuilder,
+  InlineSpinner,
+} from '@openmsupply-client/common';
+import { EncounterFragment, useEncounter } from 'packages/programs/src';
+import { getClinicianName } from '../../Patient/Encounter';
+import { AppRoute } from 'packages/config/src';
+
+interface SidePanelProps {
+  encounter: EncounterFragment;
+  onChange: (patch: Partial<EncounterFragment> & { note?: string }) => void;
+}
+
+export const SidePanel: FC<SidePanelProps> = ({ encounter, onChange }) => {
+  const [encounterNote, setEncounterNote] = useState(
+    encounter.document.data?.note ?? ''
+  );
+  const { localisedDate } = useFormatDateTime();
+  const t = useTranslation('patients');
+
+  const {
+    data: otherEncounters,
+    isError,
+    isLoading,
+  } = useEncounter.document.list({
+    filterBy: {
+      patientId: { equalTo: encounter.patient.id },
+      id: { notEqualTo: encounter.id },
```

Comment:
Don't include the current encounter

Reply:
I actually found it confusing, personally I think the list should include the current encounter (highlighted) + scheduled encounters to have the full picture in the side bar...

Reply:
For later: we should include the encounter type in the list, or filter the list by the current encounter type...

Reply:
Yeah, okay. I've changed it to only show the current encounter type, and the current encounter is displayed in **Bold**, which makes it easier to keep track when clicking between them.